space-plugins/nuxt-base/server/middleware/02.auth.global.ts

```diff
@@ -1,5 +1,8 @@
 export default defineEventHandler(async (event) => {
 	const appConfig = useAppConfig();
+	if (appConfig.appBridge.enabled) {
+		return;
```

Comment:
Would it make sense to extend this logic, to say if appBridge is enabled then it always runs verifyToken and it needs to be attached to the authorization header?

Reply:
but to only apply it to api requests

Reply:
> it always runs verifyToken and it needs to be attached to the authorization header

Interesting! For pages, we're relying on the layout, so that it can run useAppBridge() and it deals with everything. For API requests,

1. we need to get the token from sessionStorage, and attach to a HTTP request
2. the event handler on the server side validates this token first
3. and then perform its own action

Your suggestion could handle 2 and 3. But I feel like we could even have a separate middleware dedicated for validating requests for App Bridge, to avoid bloating the oauth middleware. (we should probably rename it auth -> oauth middlware)

One thing I'm still struggling to find an answer for is... 1. Users could definitely get the token and attach it to the requests, but is there a way for us to do that automatically? 🤔

Reply:
Oh I just solved 1. A commit coming soon

Reply:
https://github.com/storyblok/space-tool-plugins/pull/64/commits/1f161eaad60513e16ff2a8ec1b2b01669626d89c

what do you think? :) 

(learned it from https://github.com/nuxt/test-utils/issues/798)